### PR TITLE
Fetch events api

### PR DESCRIPTION
**Pull request by kathlynnguyenh11**

### CODE REVIEW

File: app/app.py [hunk 1]
```diff
@@ -120,6 +143,29 @@ def api_contacts():
...
+    """
+    """
+    result = db.engine.execute(
+        "SELECT * FROM CONTACTS WHERE person_id = "
+        + person_id
+        + "AND date_time = "
+        + str(date_time)
+    )
+    """
+    result = db.engine.execute(
```
Comment (peppermintpatty5) [inline app/app.py:151]: Avoid using raw SQL

Comment (kathlynnguyenh11) [reply app/app.py:151]: Just removed the line
I haven't started on the filter clause to fetch event yet

File: src/pages/CalendarView.jsx [hunk 2]
```diff
@@ -4,11 +4,11 @@ import 'react-calendar/dist/Calendar.css';
 import ActivityOption from '../components/ActivityOption';
 import DateInformation from '../components/DateInformation';
 import ContactOption from '../components/ContactOption';
+import FrequencyOption from '../components/FrequencyOption';
 
 const BASE_URL = '/api/v1/events';
-const FREQUENCY = ['Once', 'Daily', 'Weekly', 'Biweekly', 'Monthly'];
 
-function CalendarView() {
+export default function CalendarView() {
```
Comment (peppermintpatty5) [inline src/pages/CalendarView.jsx]: I prefer putting `export default CalendarView;` as a separate statement. This is especially needed for props validation.

Comment (kathlynnguyenh11) [reply src/pages/CalendarView.jsx]: That makes more sense!
I made a new commit for it

**Review by peppermintpatty5 — APPROVED** [review]

**Pull request merged**